css/common.styl

```diff
@@ -166,6 +166,8 @@ $owned-card
   overflow: hidden
   text-align: left
   text-decoration: none
+  height: 270px
```

Comment:
Since a fixed height is defined, then the max-height css property doesn't make sense anymore. It can be removed.